lib/msal-node/src/client/PublicClientApplication.ts

```diff
@@ -126,4 +162,62 @@ export class PublicClientApplication extends ClientApplication implements IPubli
         };
         return this.acquireTokenByCode(tokenRequest);
     }
+
+    /**
+     * Returns a token retrieved either from the cache or by exchanging the refresh token for a fresh access token. If brokering is enabled the token request will be serviced by the broker.
+     * @param request 
+     * @returns 
+     */
+    async acquireTokenSilent(request: SilentFlowRequest): Promise<AuthenticationResult | null> {
+        const correlationId = request.correlationId || this.cryptoProvider.createNewGuid();
+        this.logger.trace("acquireTokenSilent called", correlationId);
+
+        if (this.nativeBrokerPlugin) {
+            const brokerRequest: NativeRequest = {
+                ...request,
+                clientId: this.config.auth.clientId,
+                scopes: request.scopes || OIDC_DEFAULT_SCOPES,
+                redirectUri: `${Constants.HTTP_PROTOCOL}${Constants.LOCALHOST}`,
+                authority: request.authority || this.config.auth.authority,
+                correlationId: correlationId,
+                extraParameters: request.tokenQueryParameters,
+                accountId: request.account.nativeAccountId,
+                forceRefresh: request.forceRefresh || false
+            };
+            return this.nativeBrokerPlugin.acquireTokenSilent(brokerRequest);
+        }
+
+        return super.acquireTokenSilent(request);
+    }
+
+    /**
+     * Removes cache artifacts associated with the given account
+     * @param request 
+     * @returns 
+     */
+    async signOut(request: SignOutRequest): Promise<void> {
```

Comment:
Slightly worried if this could be mistaken for an API doing a service signout (end session). I suppose the name was chosen to align with X.. (likely lacking context, apologies)

Reply:
`signOut` API has the same name across all MSAL APIs except for`msal-browser`. What do you have in mind?

Reply:
@sameerag didn't know that! My worry is, this API only does a "local" signout, it doesn't end the session with IdP. We probably should emphasize this in API description as it's reasonable to expect this to be misunderstood.